internal/system/agent/v2/router.go

```diff
@@ -29,6 +29,7 @@ func LoadRestRoutes(r *mux.Router, dic *di.Container) {
 
 	ac := smaController.NewAgentController(dic)
 	r.HandleFunc(v2.ApiHealthRoute, ac.GetHealth).Methods(http.MethodGet)
+	r.HandleFunc(v2.ApiMetricsRoute+"/{services}", ac.GetMetrics).Methods(http.MethodGet)
```

Comment:
add a route in contract just like
https://github.com/edgexfoundry/go-mod-core-contracts/blob/2c76dd9b4bb8135d36e7dd88eb99c6fe0fcdb492/v2/constants.go#L115

try to add all SMA routes into contract